sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -220,6 +226,29 @@ internal static RecognizeLinkedEntitiesResultCollection ConvertToRecognizeLinked
 
         #endregion
 
+        #region Healthcare
+
+        internal static RecognizeHealthcareEntitiesResultCollection ConvertToRecognizeHealthcareEntitiesResultCollection(HealthcareResult results)
+        {
+            var healthcareEntititesResults = new List<DocumentHealthcareResult>();
+
+            //Read errors
+            foreach (DocumentError error in results?.Errors)
```

Comment:
why check that `results` is not null? in which cases will result be `null`?

Reply:
will remove